podcasts/New Detail/Episode Add/ManualPlaylistsChooserViewController.swift

```diff
@@ -158,15 +158,24 @@ class ManualPlaylistsChooserViewController: PCViewController {
 
         FileLog.shared.console("Added \(added), removed \(removed)")
 
+        var changedPlaylists: Set<EpisodeFilter> = []
+
         manualPlaylists.forEach { playlist in
             if added.contains(playlist.uuid) {
                 dataManager.add(episodes: [episode], to: playlist)
+                changedPlaylists.insert(playlist)
             }
             if removed.contains(playlist.uuid) {
                 dataManager.deleteEpisodes([episode.uuid], from: playlist)
+                changedPlaylists.insert(playlist)
```

Comment:
I don't think it's necessary here as the data manager here calls the `PlaylistDataManager` and the method `func deleteEpisodes(_ episodeUuids: [String], from playlist: EpisodeFilter, dbQueue: PCDBQueue)` already set the playlist as not synced:
```
playlist.syncStatus = SyncStatus.notSynced.rawValue
                try db.executeUpdate("UPDATE \(DataManager.playlistsTableName) SET syncStatus = ? WHERE uuid = ?", values: [playlist.syncStatus, playlist.uuid])
```